app/models/axis.rb

```diff
@@ -0,0 +1,11 @@
+class Axis < ApplicationRecord
```

Comment:
If we're going to use the axis' name attribute for the JS variable, should it's uniqueness be validated here? Or perhaps at the DB level?

Reply:
Not needed any more, the names will be more for the UI than anything else.